drivers/hmis_external_apis/app/assets/javascripts/external_forms.js

```diff
@@ -10,11 +11,21 @@ $(function () {
     $('#spinnerModal').modal('hide');
     form.reset();
     document.querySelector('main').remove();
+
+    // If 'Family' selected, or we just submitted a non-HoH member to an existing household,
+    // show the option to add another member in the Success modal.
+    if (window.isFamilyHoH || window.isFamilyMember) {
+      $('#addAnotherHouseholdMemberButton').show();
+    }
     $('#successModal').modal('show');
   }
 
   $('.reload-button').on('click', function () {
-    window.location.reload();
+    window.location = window.location.pathname; // drop hh_id param
+  });
+
+  $('#addAnotherHouseholdMemberButton').on('click', function () {
+    window.location = window.location.pathname + '?hh_id=' + window.householdId;
   });
```

Comment:
these don't seem to work for me since it doesn't include the origin, at least locally. These seem to work though
```js
  $('.reload-button').on('click', function () {
    let url = new URL(window.location.href);
    window.location.href = url.origin + url.pathname;
  });

  $('#addAnotherHouseholdMemberButton').on('click', function () {
    let url = new URL(window.location.href);
    url.searchParams.set('hh_id', window.householdId);
    window.location.href = url.toString();
  });
```

Reply:
Weird, it works for me (and martha) locally. What behavior are you seeing?
I will update to your suggestion

Reply:
I didn't look into it too closely but it seemed to be dropping host, sending the browser to something like:
http:///hmis_external_api/external_forms/tt-pit

I can try to replicate if it needs investigation

Reply:
No that's fine, just curious. I assume it is because you have a different local dev setup